modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/edit_workflow_task.jsp

```diff
@@ -33,11 +33,10 @@ String languageId = LanguageUtil.getLanguageId(request);
 
 WorkflowTask workflowTask = workflowTaskDisplayContext.getWorkflowTask();
 
-long classPK = workflowTaskDisplayContext.getWorkflowContextEntryClassPK(workflowTask);
-
 WorkflowHandler<?> workflowHandler = workflowTaskDisplayContext.getWorkflowHandler(workflowTask);
 
-AssetRenderer<?> assetRenderer = workflowHandler.getAssetRenderer(classPK);
+AssetRenderer<?> assetRenderer = workflowHandler.getAssetRenderer(
+	workflowTaskDisplayContext.getWorkflowContextEntryClassPK(workflowTask));
```

Comment:
@MarinhoFeliphe The `getWorkflowContextEntryClassPK` is not needed anymore. My suggestion is to remove it and just use the `workflowHandler.getEntryClassPK` instead. Can you please try this approach?

Thx man